spec.html

```diff
@@ -9214,7 +9214,7 @@ <h1>
         1. For each child node _child_ of this Parse Node, do
           1. If _child_ is an instance of _symbol_, return *true*.
           1. If _child_ is an instance of a nonterminal, then
-            1. Let _contained_ be the result of _child_ Contains _symbol_.
+            1. Let _contained_ be _child_ Contains _symbol_.
```

Comment:
I like the old form better for Contains.

Reply:
Agreed, though I'm not a huge fan of using infix notation for `Contains` in the first place.